proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,196 @@
+# Title
+
+- Author Name [Kat Marchan (@zkat)](https://github.com/zkat)
+- Start Date 2021-03-04
+- GitHub Issue (GitHub Issue link)
+- GitHub PR (GitHub PR link)
+
+## Summary
+
+This specifies the syntax and behavior for the `dotnet.exe nuget trusted-signers` command.
+
+## Motivation
+
+This is part of a larger effort to reach full parity between `nuget.exe` and
+`dotnet.exe` for NuGet-related operations.
+
+## Explanation
+
+### Functional explanation
+
+#### `dotnet nuget trusted-signers`
+
+```
+Provides the ability to manage the list of trusted signers.
+
+USAGE:
+    dotnet nuget trusted-signer [OPTIONS] [COMMAND]
+
+COMMANDS:
+    list (default)
+    trust-package <name> <package>...
+    trust-source <name> [source-url]
+    trust-certificate <name> <fingerprint>
+    remove <name>
+    sync <name>
+```
+
+#### `dotnet nuget trusted-signers list`
```

Comment:
Verb after context to match dotnet CLI guideline

Reply:
This is already in that format? Am I missing something?

Reply:
The context is nuget here. So it would be 'dotnet nuget list trusted-signers'

Reply:
see https://github.com/NuGet/Home/pull/10628#discussion_r591990207